docs/tools/vdb_table/data/Hyperspace.json

```diff
@@ -0,0 +1,180 @@
+    "comment": ""
+  },
+  "geo_search": {
+    "support": "full",
+    "source_url": "https://docs.hyper-space.io/hyperspace-docs/flows/es-dsl-query-interface/candidate-filtering#match-between-geo-coordinates",
+    "comment": ""
+  },
+  "multi_vec": {
+    "support": "full",
+    "source_url": "https://docs.hyper-space.io/hyperspace-docs/flows/setting-up/building-and-running-queries/building-a-vector-search-query",
+    "comment": "can support large number of vectors in same query"
+  },
+  "sparse_vectors": {
+    "support": "",
+    "source_url": "",
+    "comment": ""
+  },
+  "bm25": {
+    "support": "full",
+    "source_url": "https://docs.hyper-space.io/hyperspace-docs/flows/es-dsl-query-interface/candidate-score#rarity-score-tf-idf",
+    "comment": ""
+  },
+  "full_text": {
+    "support": "partial",
+    "source_url": "https://docs.hyper-space.io/hyperspace-docs/flows/data-collections/python-query-interface/candidate-filtering#exact-match-between-keywords",
+    "comment": ""
+  },
+  "embeddings_text": {
+    "support": "full",
+    "source_url": "",
+    "comment": "Embeddings by user can be integrated with python client"
+  },
+  "embeddings_image": {
+    "support": "full",
+    "source_url": "",
+    "comment": "Embeddings by user can be integrated with python client"
+  },
+  "embeddings_structured": {
+    "support": "",
+    "source_url": "",
+    "comment": ""
+  },
+  "rag": {
+    "support": "",
+    "source_url": "",
+    "comment": ""
+  },
+  "recsys": {
+    "support": "",
+    "source_url": "",
+    "comment": ""
+  },
+  "langchain": {
+    "support": "",
+    "source_url": "",
+    "comment": ""
+  },
+  "llamaindex": {
+    "support": "",
+    "source_url": "",
+    "comment": ""
+  },
+  "managed_cloud": {
+    "support": "full",
+    "source_url": "",
+    "comment": ""
+  },
+  "pricing": {
+    "value": "Per usage",
+    "source_url": "",
+    "comment": ""
+  },
+  "in_process": {
+    "support": "full",
+    "source_url": "",
+    "comment": ""
+  },
+  "multi_tenancy": {
+    "support": "none",
+    "source_url": "",
+    "comment": ""
+  },
+  "disk_index": {
+    "support": "",
+    "source_url": ""
+  },
+  "ephemeral": {
+    "support": "",
+    "source_url": "",
+    "comment": ""
+  },
+  "sharding": {
+    "support": "",
+    "source_url": "",
+    "comment": ""
+  },
+  "doc_size": {
+    "bytes": 0,
+    "unlimited": false,
+    "source_url": "",
+    "comment": ""
+  },
+  "vector_dims": {
+    "value": 8192,
+    "unlimited": false,
+    "source_url": "",
+    "comment": ""
+  },
+  "github_stars": {
+    "value": 0,
+    "source_url": "https://github.com/hyper-space-io",
+    "comment": "",
+    "value_90_days": 0
+  },
+  "docker_pulls": {
+    "value": 0,
+    "source_url": "",
+    "comment": "",
+    "value_90_days": 0
+  },
+  "pypi_downloads": {
+    "value": 0,
+    "source_url": "",
```

Comment:
it would be good to publish the python sdk as a pypi package, so that the downloads can be tracked here. It can be kept blank for now